ingestors/support/convert.py

```diff
@@ -72,17 +73,15 @@ def _document_to_pdf(self, unique_tmpdir, file_path, entity, timeout=TIMEOUT):
             file_path,
         ]
         try:
-            for attempt in range(1, CONVERT_RETRIES):
+            for attempt in range(1, settings.CONVERT_RETRIES):
```

Comment:
Nothing introduced in this PR, but I just noticed that this ”definition" of retries is inconsistent with e.g. worker retries in servicelayer (if I’m not misunderstanding something).

Here, setting `CONVERT_RETRIES=3` means that there will be max. 3 attempts in total.

In servicelayer, setting `WORKER_RETRY=3` means that there will be max. 4 attempts (one original attempt + 3 retries).

Reply:
By the way, do we actually need a separate retry mechanism for document conversions? If the conversion fails, the ingest task fails, but the ingest task itself will also be retried?

So if I’m understanding this correctly, if `WORKER_RETRY=3` and `CONVERT_RETRIES=3`, there can be up to 12 attempts at converting a file (up to 4 task-level attempts * 3 convert attempts).

Reply:
I think you are right and we should consider dropping the notion of an isolated `CONVERT_RETRIES`. How do you feel about removing it completely @tillprochaska ?

Reply:
That sounds good to me. Essentially inlining document conversions into ingest-file eliminated a whole class of problems that probably were the reason for the separate (higher) retry count.

I think it makes everything much easier to understand (for developers and admins) and if necessary we can always bring the logic back without too much effort.